pymc3/__init__.py

```diff
@@ -23,6 +23,7 @@
 from .tests import test
 
 from . import glm
+from . import gp
```

Comment:
Not sure it belongs in models actually. GP is different enough that it probably warrants its own submodule.